### DISCUSSION TITLE
Question on changing the volume directory?

### DISCUSSION THREAD

**Discussion by killmasta93** [nextcloud/all-in-one#1691] [Questions] [answered]
HI
I was wondering if its possible to change the volume directory for apache, database, and the nextcloud directory containers? currently by default its stored /var/lib/docker/volumes 
was wondering if its possible to put it another location?
Thank you

**Comment by szaimen** [comment, marked as answer]
Hi, see https://github.com/nextcloud/all-in-one#how-to-store-the-filesinstallation-on-a-separate-drive
